workflows/sample/tasks/pbsv.wdl

```diff
@@ -1,15 +1,20 @@
 version 1.0
 
-import "../structs/BamPair.wdl"
-import "./pbsv_discover.wdl" as pbsv_discover
-import "./common.wdl" as common
-import "./separate_data_and_index_files.wdl"
+#import "../../common/structs.wdl"
+#import "./pbsv_discover.wdl" as pbsv_discover
+#import "./common.wdl" as common
+#import "../../common/separate_data_and_index_files.wdl"
+
+import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/dev/workflows/common/structs.wdl"
+import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/dev/workflows/sample/tasks/pbsv_discover.wdl" as pbsv_discover
+import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/dev/workflows/sample/tasks/common.wdl" as common
+import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/dev/workflows/common/separate_data_and_index_files.wdl"
 
 task pbsv_call {
   input {
     Int threads = 8
     String region 
-    String extra = "--ccs -m 20 -A 3 -O 3"
+    String extra = "--ccs -m 20 -A 3 -O 3 -P 20"
```

Comment:
This reverted a change we specifically made to update pbsv to v2.6.0.

Reply:
Fixed.